samples/full-bundle/README.md

```diff
@@ -1,15 +1,145 @@
-# Sample
+# Sample -  Getting Started with Web Chat CDN
 
-A simple web page with a maximized and full-featured Web Chat.
+> This is a great sample for first-time Web Chat users.
 
-# How to run
+## Description
+A simple web page with a maximized and full-featured Web Chat embed from a CDN. This includes Adaptive Cards, Cognitive Services, and Markdown-It dependencies. 
 
-- Run `npx serve`
+
+# Test out the hosted sample
+- [Try out MockBot](https://microsoft.github.io/BotFramework-WebChat/full-bundle)
+
+# How to run locally
+
+- Fork and clone this repo
+- Navigate to `/Your-Local-WebChat/samples/full-bundle` in command line
+- Run `npx serve` in the full-bundle directory
```

Comment:
Is there an `npm i` that's supposed to happen before this?

Reply:
No, isn't it simple? 😝

You only need to do `npm install` on advanced route (customizing using React).